modules/serverless-sinkbinding-intro.adoc

```diff
@@ -12,3 +12,8 @@ The `SinkBinding` object injects environment variables into the `PodTemplateSpec
 
 `K_SINK`:: The URL of the resolved sink.
 `K_CE_OVERRIDES`:: A JSON object that specifies overrides to the outbound event.
+
+[NOTE]
+====
+The `SinkBinding` object currently does not support custom revision names for services.
```

Comment:
You could remove "currently" from this sentence.

Reply:
We want to imply that this will be resolved soon (that's the plan). Still no?

Reply:
Understood, thanks for clarifying that. I will leave the decision to you.